test/test-functions

```diff
@@ -741,29 +915,24 @@ install_config_files() {
     # we want an empty environment
     > $initdir/etc/environment
     > $initdir/etc/machine-id
+
+    # This is used by TEST-30-ONCLOCKCHANGE
+    inst_any /usr/share/zoneinfo/Europe/Kiev
```

Comment:
Is this necessary, as we install the exact same files, among others, in `install_zoneinfo` below (which is also called by `setup_basic_environment`?

Reply:
It was done in the original `TEST-30-*/test.sh`, so I assumed it serves a purpose ;) Gone now.